### PR TITLE
Display Parameters in Summaries

### PR DESCRIPTION
**Pull request by dubeejw**
- Adds functionality to display parameters when getting the summary for packages, actions, and triggers.

PG1: 351

### CODE REVIEW

**Comment by dubeejw** [comment]
@mdeuser, ready for review.

File: tools/go-cli/go-whisk-cli/commands/rule.go [hunk 1]
```diff
@@ -404,7 +404,7 @@ var ruleGetCmd = &cobra.Command{
         }
 
         if (flags.rule.summary) {
-            fmt.Fprintf(color.Output, "rule /%s/%s\n", rule.Namespace, rule.Name)
+            fmt.Fprintf(color.Output, "%s /%s/%s\n", boldString("rule"), rule.Namespace, rule.Name)
```
Comment (mdeuser) [inline tools/go-cli/go-whisk-cli/commands/rule.go]: @rabbah - Should the `wsk rule get RULE --summary` command output the active/inactive status with the full rule name?  Currently, only the full name is displayed.

File: tests/src/system/basic/WskBasicTests.scala [hunk 2]
```diff
@@ -105,6 +105,51 @@ class WskBasicTests
             wsk.pkg.list().stdout should include(name)
     }
 
+    it should "create, and get a package summary" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val packageName = "packageName"
+            val actionName = "actionName"
+            val packageAnnots = Map(
+                "description" -> JsString("packageDesc"),
```
Comment (mdeuser) [inline tests/src/system/basic/WskBasicTests.scala]: I recommend using spaces in the test description text

File: tests/src/system/basic/WskBasicTests.scala [hunk 3]
```diff
@@ -105,6 +105,51 @@ class WskBasicTests
...
+
+            assetHelper.withCleaner(wsk.pkg, packageName) {
+                (pkg, _) =>
+                    pkg.create(packageName, annotations = packageAnnots)
+            }
+
+            wsk.action.create(packageName + "/" + actionName, defaultAction, annotations = actionAnnots)
+
+            val stdout = wsk.pkg.get(packageName, summary = true).stdout
+            stdout should include regex("""package /.*/packageName: packageDesc\s*\(params: paramName1, paramName2\)""")
```
Comment (mdeuser) [inline tests/src/system/basic/WskBasicTests.scala]: Use variable substitution for the packageName, actionName, and description constant strings defined above.

File: tools/go-cli/go-whisk-cli/commands/rule.go (hunk 1 above)
Comment (rabbah) [inline tools/go-cli/go-whisk-cli/commands/rule.go]: If the information is available, sounds like an improvement.

**Comment by dubeejw** [comment]
@mdeuser, rule status added.

**Comment by mdeuser** [comment]
@dubeejw - Great!  Can you provide a sample output of each summary.  I'm not sure that all translatable strings are externalized..

**Comment by dubeejw** [comment]
```
package /namespace/pkgName: packageDesc
   (params: paramName, paramName2)
 action /namespace/pkgName/act1: A description
   (params: aKey, anotherKey)
```

**Comment by dubeejw** [comment]
```
trigger /jwdubee@us.ibm.com_dev/triggerTest: A description
   (params: aKey, anotherKey)
```

**Comment by dubeejw** [comment]
```
rule /jwdubee@us.ibm.com_dev/kjjhjh
   (status: inactive)
```

**Comment by dubeejw** [comment]
@mdeuser, example outputs provided above.

**Comment by dubeejw** [comment]
@csantanapr, please merge if @mdeuser doesn't have anymore comments.

**Comment by dubeejw** [comment]
The "status" message is now externalized.

**Comment by mdeuser** [comment]
@dubeejw - Was thinking about the following output
```
package /namespace/pkgName: packageDesc
   (params: paramName, paramName2)
 action /namespace/pkgName/act1: A description
   (params: aKey, anotherKey)
```
Ugh.. This might affect test cases...but, IMHO, I think `params` should be externalized as `parameters`

**Comment by dubeejw** [comment]
@mdeuser, the "params" string is now externalized as "parameters".

**Comment by mdeuser** [comment]
:+1:

**Comment by dubeejw** [comment]
@csantanapr, ready for merge.

**Comment by csantanapr** [comment]
there was a conflict, tried to fix and running PG2 134 with
https://github.com/openwhisk/openwhisk/compare/master...csantanapr:pr-1121

**Comment by csantanapr** [comment]
@dubeejw Could you rebase this PR and resubmit a PG, it has a conflict and there have some changes in go-cli code base.
sorry for the trouble.

**Comment by dubeejw** [comment]
PG1 430

**Comment by csantanapr** [comment]
merged 4d9b7c6f40e736bb0c85d712654dc82e90e602fb

**Pull request closed without merging**